website/source/docs/commands/apply.html.markdown

```diff
@@ -53,10 +53,11 @@ The command-line flags are all optional. The list of available flags are:
   multiple times.
 
 * `-var 'foo=bar'` - Set a variable in the Terraform configuration. This
-  flag can be set multiple times.
+  flag can be set multiple times. Variable values are interpreted as HCL, so
```

Comment:
Believe it or not, it is! It's not widely known, but I've definitely encountered people who use it. (some adopted it because it was an option through the viper config package)